source/mobile/mobile-compile-yourself.rst

```diff
@@ -33,3 +33,17 @@ To deploy in production with privately-hosted mobile apps compiled by your organ
 .. image:: ../images/mobile_push_contents.png
 
 6. Deploy to an Enterprise App Store
+
+Commands to Build the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The set of commands for building the app are used in conjunction with Fastlane.  For most of them, you will need to modify the Fastfile in order to make it work since they are very tightly coupled with Mattermost's build and deployment process.
+
+You will **always** be able to build an unsigned version of the app as it does not need provisioning profiles or certificates as long as you set up *Fastlane* in your environment.
```

Comment:
We usually reserve bolding for [field names](https://docs.mattermost.com/process/documentation-guidelines.html#text-formatting), so we can remove it for `always`. Also, `Fastlane` should either be italicized throughout the section or not for consistency.

We can also shorten this sentence a little, maybe:

```
If you set up Fastlane, no provisioning profiles or certificates are needed to build an unsigned version of the app.
```

^should make sure the content stays the same after rephrasing.